meta-mender-qemu/recipes-kernel/linux/linux-yocto_%.bbappend

```diff
@@ -1,14 +1,15 @@
 FILESEXTRAPATHS_prepend := "${THISDIR}/${PN}:"
 
 # Use custom defconfig in order to enable use of the vexpress model.
-SRC_URI_append_vexpress-qemu = " file://defconfig \
+SRC_URI_append_vexpress-qemu = " file://vexpress-qemu-mender.cfg \
```

Comment:
I don't know why, but this change makes one test when running `losetup --show -f /data/tmp-file-system`:
```
losetup: cannot find an unused loop device
```

Does it ring a bell? Have we accidentally changed something in the configuration?

Reply:
Well CONFIG_BLK_DEV_LOOP is required: https://github.com/mendersoftware/meta-mender/commit/481bfc9abfcdac83447a1fc67a8342847b75363a

Not sure why diffconfig missed this. But I just reworked this patch to simply remove the bits that were generating warnings.  We still use a defconfig which would Ideally be exchanged for a config fragment but I had no luck making that happend.